build-system/tasks/storybook-react/react-env/main.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable local/no-forbidden-terms */
```

Comment:
What forbidden terms are used in this file? Prefer `eslint-disable-next-line` or adding this file to the forbidden-terms allowlist for the rule it's violating (usually preferred)

Reply:
I forgot why I added that there, but it doesn't appear to be needed anymore.

Reply:
If i recall, there was a linting rule that prevented me from using the `@storybook/addon-knobs` but it looks like I can now.